src/extensions/extra/ImageUrlCodeHints/main.js

```diff
@@ -0,0 +1,874 @@
+/*
```

Comment:
uhhh I think its:
https://github.com/humphd/brackets/pull/284/files#diff-89f6329f8b7d2cdbba386333776b999fR233 to 236
https://github.com/humphd/brackets/pull/284/files#diff-89f6329f8b7d2cdbba386333776b999fR584 to 589

I don't think there's anything else, but I would ask @sedge and @JordanTheriault  to confirm